extensions/windows-event-log/Bookmark.cpp

```diff
@@ -48,7 +48,7 @@ Bookmark::Bookmark(const std::wstring& channel, const std::wstring& query, const
   }
   const utils::ScopeGuard guard_hEventResults([hEventResults]() { EvtClose(hEventResults); });
 
-  if (!EvtSeek(hEventResults, 0, 0, 0, EvtSeekRelativeToLast)) {
+  if (!EvtSeek(hEventResults, 0, 0, 0, processOldEvents? EvtSeekRelativeToFirst : EvtSeekRelativeToLast)) {
```

Comment:
What happens in case the property is true and the server restarted? (It processed some events before, so not all event should be reprocessed)

Reply:
Since already processed events are bookmarked, only events which are not processed from  bookmark point will be processed, it could be not yet processed old events or new ones.